views/index.pug

```diff
@@ -3,6 +3,36 @@ extends layout
 block content
   h1= title
   p Welcome to #{title}
+
+  span#start.target
+
+  span#about.target
+
+  .modal
+    .content.vertical-align-middle
+      form.createPost(action='/create' method='POST')
+        header
+          h1 Create new post
+        .formFields
+          label Title
+          input.createInput(type='text', name='title', value='title')
```

Comment:
Is the `createInput` class necessary?  You could just target the `input` and `textarea` children of this form.